components/FrontPage/Combined/frontImage.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Image from "next/image";
+
+export default function FrontImage() {
+  return (
+    <div className="flex w-full flex-col items-center gap-12">
+      {/* Desktop light */}
+      <Image
+        className="hidden w-[85vw] max-w-[1000px] sm:block dark:sm:hidden"
+        width={1200}
+        height={900}
+        src={"/Images/image1.png"}
+      ></Image>
```

Comment:
Please prefer  tag instead of next/image because Next's Image tag consumes a lot of bandwidth and can be costly.